### PR TITLE
LKE-7924 alerts config maxConcurrencyLimit & maxMatchesLimit

### PR DESCRIPTION
**Pull request by mzorkany**
TO BE MERGED in v4.0.2

This PR is a part of the alerts enhancement topic
The alerts configs are modified by: 
Removing the maxConcurrencyLimit 
Renaming maxCasesLimit to maxMatchesLimit

https://linkurious.atlassian.net/browse/LKE-7924
 https://linkurious.atlassian.net/browse/LKE-7925

Server PR:
https://github.com/Linkurious/linkurious-server/pull/3791

### CODE REVIEW

File: src/api/Config/types.ts [hunk 1]
```diff
@@ -202,9 +202,8 @@ export interface IAzureSearchConfig extends IVendorConfig {
 
 export interface IAlertsConfig {
   enabled?: boolean;
-  maxCasesLimit?: number;
+  maxMatchesLimit?: number;
```
Comment (davidrapin) [inline src/api/Config/types.ts:205]: @mzorkany did you try it locally? You are missing the config updates.

**Review by davidrapin — CHANGES_REQUESTED** [review]

File: src/api/Config/types.ts (hunk 1 above)
Comment (mzorkany) [reply src/api/Config/types.ts:205]: @davidrapin 
Sorry i forgot to mention in the description the server PR
https://github.com/Linkurious/linkurious-server/pull/3791

Comment (davidrapin) [reply src/api/Config/types.ts:205]: why not move this change to the #3791 PR? these two PRs don't seem to make sense separately

Comment (davidrapin) [reply src/api/Config/types.ts:205]: oh nevermind, different project, sorry

**Review by davidrapin — APPROVED** [review]

**Review by gmasclet — APPROVED** [review]

**Pull request merged**